tests/test_integration.py

```diff
@@ -792,3 +794,46 @@ def test_ajax_refresh(self):
         )
         self.assertNotIn("/ajax/", history_panel.text)
         self.assertIn("/json_view/", history_panel.text)
+
+
```

Comment:
```suggestion

from django.test import AsyncRequestFactory

```

Reply:
Thank you for the tip

Reply:
You must move that line up near the top of the file with the other imports.